sycl/include/sycl/detail/pi.h

```diff
@@ -1369,6 +1173,11 @@ typedef struct {
 
 using pi_launch_property = _pi_launch_property;
 
+struct sycl_device_binaries_struct;
```

Comment:
Strictly speaking, a couple of our compiler interfaces are part of the PI interface. But it won't be the case with UR if I understand correctly and therefore I went with those typedefs. There is also a few places (pi2ur, one unit-test and program manager) where reinterpret casts appeared to convert between `pi` alias and an actual type.

Reply:
UR won't depend on these types.